src/org/jetbrains/plugins/scala/caches/HydraArtifactsCache.scala

```diff
@@ -0,0 +1,43 @@
+package org.jetbrains.plugins.scala.caches
+
+import java.util.concurrent.ConcurrentHashMap
+
+import com.intellij.openapi.diagnostic.Logger
+import org.jetbrains.plugins.scala.project.Platform
+import org.jetbrains.plugins.scala.project.template.Downloader
+
+/**
+  * @author Maris Alexandru
+  */
+object HydraArtifactsCache {
+  private val LOG = Logger.getInstance(this.getClass)
+  private val cache = new ConcurrentHashMap[(String, String), Seq[String]]()
+  private val SplitRegex = "\\[info\\] \\* Attributed\\(|\\)"
+  private val ArtifactsRegex = "\\[info\\] \\* Attributed\\(.*\\)"
```

Comment:
The above two regex definition should be compiled, i.e., add `.r` after the string.

Reply:
Also, is it relevant to match the `[info]` part? If not, I think it's better to remove it, as it will make our code more resilient to changes in the output produced by sbt